lib/wallet/http.js

```diff
@@ -764,9 +765,40 @@ class HTTP extends Server {
     this.get('/wallet/:id/tx/history', async (req, res) => {
       const valid = Validator.fromRequest(req);
       const acct = valid.str('account');
-      const txs = await req.wallet.getHistory(acct);
+      const reverse = valid.bool('reverse', false);
+      const limit = valid.u32('limit', 10);
+      const after = valid.brhash('after');
+      const date = valid.str('date');
```

Comment:
To reviewers: Should we parse the date here or have the date and time always be accepted as seconds?

Reply:
Gone with just the `time` argument for simplicity.